docs/build_steps.rst

```diff
@@ -26,6 +26,7 @@ of empty container):
 * :step:`SubConfig`
 * :step:`Container`
 * :step:`Tar`
+* :step:`DockerImage`
```

Comment:
Should this option really use the "Docker" branding in the name? It'd be confusing for the Podman users as well for anything else coming later. I saw that the tests seem to refer to `buildah` and the OCI Images. So maybe just play with OCI instead of specific brand names?